Firmware/cmdqueue.cpp

```diff
@@ -483,6 +484,7 @@ void get_command()
         if ((*cmd_start == 'G') && (GetPrinterState() != PrinterState::IsSDPrinting)) {
             usb_timer.start();
             SetPrinterState(PrinterState::IsHostPrinting); //set printer state busy printing to hide LCD menu while USB printing
+            eeprom_update_byte((uint8_t*)EEPROM_UVLO, PowerPanic::NO_PENDING_RECOVERY);
```

Comment:
I didn't add an if statement to check if the EEPROM value is currently st as the eeprom_update should not write to eeprom, just uses few cycles, right?